### ISSUE TITLE
Add processing times to machines - enhancement

### ISSUE THREAD

**Issue by mathwiz617** [whizzball1/The-Package-Mod#5]
In Factorio, which I see you're trying to emulate the feel of, every recipe has a crafting time. I'm thinking, perhaps add support for crafting times (in ticks) for packages and research. This would allow pack makers to add incentive for going further down a "research tree" other than more efficient recipes (that might not save enough resources to pay for the research in any short time).

It would be like upgrading assembly machines in Factorio, but without adding more blocks to the mod.

**Comment by whizzball1** [comment]
Makes sense! When I was first writing this mod I wavered between adding processing times or not; I thought it would end up with modpack makers creating pointless time gates. But for the Factorio experience and with a wise pack developer I can absolutely see it working. I'll make sure to add this when I can.
